packages/typespec-ts/test/unit/clientUtil.spec.ts

```diff
@@ -83,66 +83,25 @@ describe("client utils get rlc clients", () => {
     
     @service
     namespace MyService {
-      @client({
-        name: "MyClient",
-        service: MyService
-      })
       interface MyInterface {
         op1(): void
       }
     }
     
     @service
     namespace MySecondService {
-      @client({
-        name: "MySecondClient",
-        service: MySecondService
-      })
       interface MySecondInterface {
         op1(): void
       }
     }
-    `;
-    const clients = await getRLCClientsFromTypeSpec(content);
-    assert.equal(clients.length, 2);
-    assert.equal(clients[0]?.name, "MyServiceClient");
-    assert.equal(clients[1]?.name, "MySecondServiceClient");
-  });
-
-  it("should handle both 1:1 and 1:N mappings between @service and @client", async () => {
-    const content = `
-    import "@azure-tools/typespec-client-generator-core";
-
-    using Azure.ClientGenerator.Core;
     
-    @service
-    namespace MyService {
-      @client({
-        name: "MyClient",
-        service: MyService
-      })
-      interface MyInterface {
-        op1(): void
-      }
-    }
-    
-    @service
-    namespace MySecondService {
-      @client({
-        name: "MySecondClient",
-        service: MySecondService
-      })
-      interface MySecondInterface {
-        op1(): void
+    @client(
```

Comment:
Can not define multiple explicit clients with multiple services. https://github.com/Azure/typespec-azure/pull/3617/files